pkg/tenant/config/config.go

```diff
@@ -58,6 +63,14 @@ type QuarterdeckConfig struct {
 	WaitForReady time.Duration `default:"5m" split_words:"true"`
 }
 
+// Configures the client connection to Ensign.
```

Comment:
Is `split_words:"true"` required for `Endpoint` and `Insecure`?

Reply:
Apparently not, thanks!